app/frontend/styles/components/flashes.scss

```diff
@@ -63,3 +67,11 @@
     color: govuk-colour('red');
   }
 }
+
+.alert {
+  background-image: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="21" height="20" viewBox="0 0 21 20"><path fill-rule="evenodd" clip-rule="evenodd" d="M20.3423 9.85217C20.3423 15.2057 15.9389 19.5493 10.5114 19.5493C5.08403 19.5493 0.680542 15.2057 0.680542 9.85217C0.680542 4.4986 5.08403 0.155029 10.5114 0.155029C15.9389 0.155029 20.3423 4.4986 20.3423 9.85217ZM9.1855 3.47218L9.4223 11.6886H11.6164L11.8445 3.47218H9.1855ZM8.97469 15.0479C8.88707 14.8457 8.8429 14.6293 8.8429 14.3986C8.8429 14.1736 8.88707 13.9607 8.97469 13.7622C9.06231 13.5636 9.18397 13.3886 9.33893 13.2386C9.4939 13.0886 9.67348 12.97 9.87841 12.8836C10.0833 12.7972 10.3028 12.7536 10.5367 12.7536C10.7648 12.7536 10.9806 12.7972 11.1819 12.8836C11.3832 12.97 11.5606 13.0886 11.7127 13.2386C11.8647 13.3886 11.9849 13.5629 12.0726 13.7622C12.1602 13.9615 12.2043 14.1736 12.2043 14.3986C12.2043 14.6293 12.1602 14.8465 12.0726 15.0479C11.9849 15.2493 11.8647 15.4257 11.7127 15.5757C11.5606 15.7257 11.3839 15.8443 11.1819 15.9307C10.9798 16.0172 10.7648 16.0607 10.5367 16.0607C10.3028 16.0607 10.0826 16.0172 9.87841 15.9307C9.67421 15.8443 9.4939 15.7257 9.33893 15.5757C9.18397 15.4257 9.06231 15.25 8.97469 15.0479Z" ></path></svg>');
```

Comment:
whilst its fine for this piece of work, i think things like this we should use https://fontawesome.com/icons?d=gallery in future if possible. seems several other DFE team already do and i have started using it in my existing PR for filters. Chris has also referenced it a few times with me as something he uses as reference for standard icons

Reply:
I use FontAwesome all the time, very happy to use them here. I thought he was making the SVG files himself 😂

Reply:
maybe ive blown his cover 😄  its certainly come up in our convos a few times